lib/OpenQA/WebAPI/Controller/Admin/Needle.pm

```diff
@@ -130,7 +130,6 @@ sub delete {
     my @removed_ids;
     for my $p (@{$self->every_param('id[]')}) {
         if (!$self->app->db->resultset('Needles')->find($p)->remove($self->current_user)) {
-            $self->stash(error => "Error removing $p");
```

Comment:
because there is no stash for error. this route especially is an AJAX route, which doesn't look at stashes